tools/scaffold/scripts/schema-scaffold.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
```

Comment:
considered this option, preferred to define the script within a folder that provides more context about its purpose rather than detail of how it is implemented.

Reply:
I think that using multiple folder for scripts used in the makefile is confusing

Reply:
👍 will move over to script folder, did not have a strong preference here